drivers/adc/ad400x/ad400x.c

```diff
@@ -195,9 +195,9 @@ int32_t ad400x_init(struct ad400x_dev **device,
 	if (ret < 0)
 		goto error;
 
+	dev->dev_id = init_param->dev_id;
```

Comment:
this changes does not look related to this commit

Reply:
ok,